src/main/java/pl/akademia/api/model/User.java

```diff
@@ -0,0 +1,32 @@
+package pl.akademia.api.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Data
+@Entity
+@Table(name = "user")
+public class User {
```

Comment:
jesli mozesz zamien User na UserApp. gdyż dodajac biblioteke Spring Security ona ma juz User i bedzie to mylace, a musimy z tego skorzystac